__tests__/BonusNumberTest.js

```diff
@@ -0,0 +1,20 @@
+import BonusNumber from '../src/BonusNumber.js';
+
+describe('보너스 번호 클래스 테스트', () => {
+  test('빈 값을 입력했을 때 예외처리', () => {
+    expect(() => new BonusNumber('', [1, 2, 3, 4, 5, 6])).toThrow('[ERROR]');
+  });
+
+  test('숫자가 아닌 문자열이 포함되어 있을 경우 예외처리', () => {
+    expect(() => new BonusNumber('abc', [1, 2, 3, 4, 5, 6])).toThrow('[ERROR]');
+  });
+
+  test('숫자의 범위가 1 ~ 45이상 넘어갈 경우 예외처리', () => {
+    expect(() => new BonusNumber(46, [1, 2, 3, 4, 5, 6])).toThrow('[ERROR]');
+    expect(() => new BonusNumber(0, [1, 2, 3, 4, 5, 6])).toThrow('[ERROR]');
+  });
+
+  test('앞에 입력 받은 숫자와 중복 될 경우 예외처리', () => {
+    expect(() => new BonusNumber(5, [1, 2, 3, 4, 5, 6])).toThrow('[ERROR]');
+  });
```

Comment:
`test.each()` 사용해서 묶어주셔도 좋을 듯하네용! 나머지 테스트 코드에서도 이렇게 묶어주시면 훨씬 코드가 깔끔해져서 보기 좋을 것 같습니다 :)
```suggestion
test.each([
    ['빈 값을 입력했을 때', '', [1, 2, 3, 4, 5, 6]],
    ['숫자가 아닌 문자열이 포함된 경우', 'abc', [1, 2, 3, 4, 5, 6]],
    ['숫자가 45를 초과할 경우', 46, [1, 2, 3, 4, 5, 6]],
    ['숫자가 1 미만일 경우', 0, [1, 2, 3, 4, 5, 6]],
    ['보너스 번호가 중복될 경우', 5, [1, 2, 3, 4, 5, 6]],
  ])('%s 예외처리', (_, input, lottoNumbers) => {
    expect(() => new BonusNumber(input, lottoNumbers)).toThrow('[ERROR]');
  });
```

Reply:
앗! 이번 과제에 적용해보겠습니다 감사해요~~!